contrib/unit_test_coverage/generate_coverage_report.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# --- generate_coverage_report.sh ---
+#
+# This script generates a coverage report for the Orbit repository. It uses 
+# llvm-cov and requires a build with build arguments -fprofile-instr-generate 
+# and -fcoverage-mapping. 
+#
+# usage: generate_coverage_report.sh SOURCE_DIR BUILD_DIR OUTPUT_DIR
+#
```

Comment:
Can you remove that #, or add a new line. Afterwards you have the constants of that file rather then a description of the script.